code/modules/chemistry/tools/syringes.dm

```diff
@@ -116,6 +116,16 @@
 					boutput(user, SPAN_ALERT("The [src.name] is full."))
 					return
 
+				if (istype(target,/obj/item/reagent_containers/synthflesh_pustule))
+					var/obj/item/reagent_containers/synthflesh_pustule/P = target
+					P.reagents.trans_to(src, src.amount_per_transfer_from_this)
+					logTheThing(LOG_CHEMISTRY, user, "draws 5 units of reagents from [constructTarget(target,"combat")] [log_reagents(target)] with a syringe [log_reagents(src)] at [log_loc(user)].")
+					if (!P.poked)
+						P.poked = TRUE
+						SPAWN(3 SECONDS)
+							P.try_healpoke()
```

Comment:
Is the intent you have 3 seconds to use the hole?

Reply:
Yes. Giving the player 3 seconds until it closes and they need to poke a new one.